js/FuzzyTimeline/FuzzyTimelineRangeSlider.js

```diff
@@ -40,35 +40,56 @@ function FuzzyTimelineRangeSlider(parent) {
 	this.sliderParentTable = this.parent.gui.sliderTable;
 	var headerRow = $("<tr></tr>");
 	var controlsRow = $("<tr></tr>");
+	var strHtml = '';
 	$(this.sliderParentTable).append(headerRow).append(controlsRow);
 
-	headerRow.append("<td>Time start</td>");
-	this.rangeStart = document.createElement("select");
-	controlsRow.append($("<td></td>").append(this.rangeStart));
+	// At special use cases, timeStart  isn't necessary
+	// construct GeoTemConfig.getString("timeStart") used for multi language
+	if (!exhibition && !forEmbeddedUse) {
+		var strHtml = "<td>" + GeoTemConfig.getString("timeStart") + "</td>";
+		headerRow.append(strHtml);
+		//headerRow.append("<td>Time start</td>");
+		this.rangeStart = document.createElement("select");
+		controlsRow.append($("<td></td>").append(this.rangeStart));
+	}
 
-	headerRow.append("<td>Time unit</td>");
+	var strHtml = "<td>" + GeoTemConfig.getString("timeUnit") + "</td>";
+	headerRow.append(strHtml);
+	//headerRow.append("<td>Time unit</td>");
 	this.rangeDropdown = document.createElement("select");
 	controlsRow.append($("<td></td>").append(this.rangeDropdown));
 
-	headerRow.append("<td>Scaling</td>");
-	this.scalingDropdown = document.createElement("select");
-	controlsRow.append($("<td></td>").append(this.scalingDropdown));
-	$(this.scalingDropdown).append("<option>normal</option>");
-	$(this.scalingDropdown).append("<option>logarithm</option>");
-	$(this.scalingDropdown).append("<option>percentage</option>");
-	$(this.scalingDropdown).change(function(eventObject){
-		var scaleMode = $(rangeSlider.scalingDropdown).find("option:selected").text();
-		rangeSlider.parent.changeScaleMode(scaleMode);
-	});
+	if (!exhibition) {
```

Comment:
Can we leave a (!exibition) here? Programmatically it seems to be ok, maybe a (exibition) would be better?